tests/dataframe/test_groupby_pytest.py

```diff
@@ -106,10 +106,18 @@ def test_groupby_aggs_mad_var_std(self, pd_agg, dropna):
         pd_flights = self.pd_flights().filter(self.filter_data)
         oml_flights = self.oml_flights().filter(self.filter_data)
 
-        pd_groupby = getattr(pd_flights.groupby("Cancelled", dropna=dropna), pd_agg)()
+        if pd_agg == "mad":
+            pd_groupby = pd_flights.groupby("Cancelled", dropna=dropna).agg(
+                lambda x: (x - x.mean()).abs().mean()
```

Comment:
I see this lambda get used in other places, could we possible have a util class that dispatches theses common function names?

Reply:
Fixed